resources/liquibase.yaml

```diff
@@ -1,3 +1,3 @@
 databaseChangeLog:
-  - includeAll:
-      path: "migrations/"
+  - include:
+      file: "migrations/000_migrations.yaml"
```

Comment:
`includeAll` resulted in all the SQL files in `resources/` to be run automatically by liquibase regardless of the DB type. I've changed it to just include the `000_migrations.yaml` file automatically, and the migrations file will pull in the rest of the SQL via `sqlFile` change types.